### PR TITLE
doc: Doc fixes for 2.8

### PR DESCRIPTION
**Pull request by umapraseeda**
Doc fixes for 2.8 - release task
NCSDK- 29833

### CODE REVIEW

File: doc/nrf/links.txt [hunk 1]
```diff
@@ -218,8 +214,6 @@
 
 .. _`Azure SDK for Embedded C`: https://azure.github.io/azure-sdk-for-c/
 
-.. _`Memfault WebBluetooth Client`: https://memfault.github.io/web-ble-example/
-
```
Comment (umapraseeda) [inline doc/nrf/links.txt:222]: Duplicate entry

File: doc/nrf/links.txt [hunk 2]
```diff
@@ -104,8 +102,6 @@
 .. _`HIDAPI releases`: https://github.com/libusb/hidapi/releases
 .. _`pyhidapi Python wrapper`: https://github.com/apmorton/pyhidapi
 
-.. _`nrf9160dk_nrf9160_partition_conf.dts`: https://github.com/nrfconnect/sdk-zephyr/blob/main/boards/arm/nrf9160dk_nrf9160/nrf9160dk_nrf9160_partition_conf.dts
```
Comment (umapraseeda) [inline doc/nrf/links.txt:107]: unused entry

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_customize_qsg.rst [hunk 3]
```diff
@@ -95,7 +95,7 @@ This can be done by changing the class and vendor names using Kconfig options (f
    CONFIG_SUIT_MPI_APP_LOCAL_1_VENDOR_NAME="ACME Corp"
    CONFIG_SUIT_MPI_APP_LOCAL_1_CLASS_NAME="Light bulb"
 
-After rebuiling you will be able to find the following part in the :file:`./build/DFU/application.yaml` file.
+After rebuilding you will be able to find the following part in the :file:`./build/DFU/application.yaml` file.
```
Comment (peknis) [inline doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_customize_qsg.rst:98]:
```suggestion
After rebuilding you can find the following part in the :file:`./build/DFU/application.yaml` file.
```

File: doc/nrf/libraries/dfu/suit_dfu.rst [hunk 4]
```diff
@@ -7,7 +7,7 @@ Software Updates for Internet of Things (SUIT)
    :local:
    :depth: 2
 
-The Software Updates for Internet of Things (SUIT) Device Firmware Upgrade (DFU) library provides functionality to a local domain for for orchestrating the update based on the SUIT manifest.
+The Software Updates for Internet of Things (SUIT) Device Firmware Upgrade (DFU) library provides functionality to a local domain for orchestrating the update based on the SUIT manifest.
```
Comment (peknis) [inline doc/nrf/libraries/dfu/suit_dfu.rst:10]:
```suggestion
The Software Updates for Internet of Things (SUIT) Device Firmware Upgrade (DFU) library provides functionality to a local domain for managing the update based on the SUIT manifest.
```

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by b-gent — APPROVED** [review]

**Review by annwoj — APPROVED** [review]

**Review by divipillai — APPROVED** [review]

**Review by richabp — APPROVED** [review]

**Review by wiba-nordic — APPROVED** [review]

**Review by nordicjm — APPROVED** [review]
Sysbuild/XIP/build parts OK

**Review by melwee01 — APPROVED** [review]

**Review by frkv — APPROVED** [review]
LGTM (TF-M, crypto drivers and and security)

**Pull request merged**